.travis.yml

```diff
@@ -67,25 +23,21 @@ before_install:
     - wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
     - chmod +x miniconda.sh
     - ./miniconda.sh -b
-    - export PATH=/home/travis/miniconda/bin:$PATH
```

Comment:
this switch is not needed.

Reply:
You mean I should keep it as `miniconda` not `miniconda2`?

Reply:
yes - there is no difference between miniconda 2 and 3. just leave as is. The rest looks really good.

Reply:
Yes you are right there is no difference between two in use as mentioned in [miniconda docs](https://docs.conda.io/en/latest/miniconda.html):
> There are two variants of the installer: Miniconda is Python 2 based and Miniconda3 is Python 3 based. Note that the choice of which Miniconda is installed only affects the root environment. Regardless of which version of Miniconda you install, you can still install both Python 2.x and Python 3.x environments.

So `wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh` actually installs minconda for python2, but directory where it is installed (the root directory containing miniconda package) is named as `miniconda2` not `miniconda`.
I checked this by making this suggested [change](https://github.com/jaladh-singhal/wsynphot/commit/758f3ab257f6c4fbfa99b4e085cd9ae23893a7e1) in my fork repo, but it leads to [build error](https://travis-ci.org/jaladh-singhal/wsynphot/builds/521804909). 
Further, I verified reason from the bash file `http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh` we are using, it defines variable:
```bash
PREFIX=$HOME/miniconda2
```
which is being used to create the root directory of miniconda for installing it. 
Thus we need to keep it `miniconda2` in `PATH`.

Reply:
okay can you make it install in `$HOME/miniconda`

Reply:
That is shell file provided by Anaconda community to handle installation of miniconda. Should we really change it?

Could you please elaborate what exactly do we want to achieve by keeping it `miniconda` so that I can understand the exact problem for finding suitable solution?

Reply:
so essentially miniconda2/3 do not matter (only the core environment). The problem is that many people when they see `miniconda2` they believe the code must be python2. The next problem is that when we switch `miniconda` we do not want to update the paths. You are right that screwing with the file itself is a bad idea - but I'm sure there are command line options for that.

Reply:
OK I'm getting it.
I'm exploring command line arguments of miniconda.sh, will make a commit once I figure it out.